centreon/tests/e2e/cypress/support/Commands.ts

```diff
@@ -112,9 +112,24 @@ Cypress.Commands.add('getIframeBody', (): Cypress.Chainable => {
 });
 
 Cypress.Commands.add(
-  'getFormFieldByIndex',
-  (rootItemNumber: number): Cypress.Chainable => {
-    return cy.getIframeBody().find('#Form').find('tr').eq(rootItemNumber);
+  'handleRequestOnDatabase',
```

Comment:
`requestOnDatabase` would be a better name, don't you think?

Reply:
What do you think of "runQueryOnDatabaseFromContainer" ?